lib/seahorn/BmcPass.cc

```diff
@@ -362,6 +362,43 @@ class BmcPass : public llvm::ModulePass {
       }
     }
 
+    // -- compute dependence of sea.assert.if()
+    assumeFn = m->getFunction("sea.assert.if");
+    if (assumeFn) {
+      for (auto *u : assumeFn->users()) {
+        if (auto *CI = dyn_cast<CallInst>(u)) {
+          CallSite CS(CI);
+          if (CS.getCaller() != &F)
+            continue;
+          dfCoi.analyze(*CI);
+        }
+      }
+    }
+    // -- compute dependence of verifier.assert.not()
+    assumeFn = m->getFunction("verifier.assert.not");
+    if (assumeFn) {
```

Comment:
this did not change. still called `assumeFn`. I think this should be factored out into a method that takes a pointer to `llvm::Function`, and if the pointer is not null computes its dependence. Then the method is called 3 times in here.

Reply:
Fixed

Reply:
We anyway wanted to get away from using strings to using defined enums... can this be in the next PR? Added in project task.

Reply:
refactored